### PR TITLE
FUT035Z+

### PR DESCRIPTION
**Pull request by Luligu**
Added FUT035Z+ with reporting since it has an external push button for onOff and brightness.
Phot and md file done in zigbee2mqtt.io.

### CODE REVIEW

File: src/devices/miboxer.ts [hunk 1]
```diff
@@ -12,6 +13,20 @@ const definitions: Definition[] = [
         vendor: 'MiBoxer',
         extend: tuya.extend.light_onoff_brightness_colortemp_color({colorTempRange: [153, 500]}),
     },
+    {
+        fingerprint: [{modelID: 'TS0502B', manufacturerName: '_TZB210_lmqquxus'}],
```
Comment (Koenkk) [inline src/devices/miboxer.ts:17]: Please merge the changes with `TS0502B` instead of adding a new definition.

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/2340

File: src/devices/miboxer.ts (hunk 1 above)
Comment (Luligu) [reply src/devices/miboxer.ts:17]: Hi, 
sorry but I got lost. Do you want me to modify this entire definition in tuya.ts that includes other 8 devices?
    {
        zigbeeModel: ['TS0502B'],
        model: 'TS0502B',
        vendor: 'TuYa',
        description: 'Light controller',
        whiteLabel: [
            tuya.whitelabel('Mercator Ikuü', 'SMI7040', 'Ford Batten Light', ['_TZ3000_zw7wr5uo']),
            {vendor: 'Mercator Ikuü', model: 'SMD9300', description: 'Donovan Panel Light'},
            tuya.whitelabel('Aldi', 'F122SB62H22A4.5W', 'LIGHTWAY smart home LED-lamp - filament', ['_TZ3000_g1glzzfk']),
            tuya.whitelabel('MiBoxer', 'FUT035Z', 'Dual white LED controller',
                ['_TZ3210_frm6149r', '_TZ3210_jtifm80b', '_TZ3210_xwqng7ol']),
            tuya.whitelabel('Lidl', '14156408L', 'Livarno Lux smart LED ceiling light', ['_TZ3210_c2iwpxf1']),
        ],
        extend: tuya.extend.light_onoff_brightness_colortemp({colorTempRange: [153, 500], noConfigure: true}),
        configure: async (device, coordinatorEndpoint, logger) => {
            device.getEndpoint(1).saveClusterAttributeKeyValue('lightingColorCtrl', {colorCapabilities: 16});
        },
    },
The other devices seem very different. Are all of these the same device internally? Push button included?

**Comment by Luligu** [comment]
Hi,
should I modify this entire definition in tuya.ts that includes other 8 devices?

File: src/devices/miboxer.ts (hunk 1 above)
Comment (Koenkk) [reply src/devices/miboxer.ts:17]: These are all led controllers with a push button, please merge them.

**Comment by Luligu** [comment]
Done.
I left:
            device.getEndpoint(1).saveClusterAttributeKeyValue('lightingColorCtrl', {colorCapabilities: 16});
but to be honest I don't know what it is but _TZB210_lmqquxus (the only device of the list I can test with) works correctly 100%.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**